attributes/default.rb

```diff
@@ -3,6 +3,5 @@
 node.default[:cyclesafe_chef][:group] = 'cyclesafe'
 node.default[:cyclesafe_chef][:directory] = '/srv/cyclesafe'
 node.default[:cyclesafe_chef][:requirements_file] = 'prod_requirements.txt'
-node.default[:cyclesafe_chef][:repository] = 'https://github.com/zemadi/CycleSafe_deploy.git'
```

Comment:
This change prevents a default deployment. The value of this attribute will register as "nil" and therefore the application provider will not run, and the django app and its associated configurations will not be affected.